exampleSite/content/dev/resource-fallthrough/fragment/footer/index.md

```diff
@@ -1,14 +1,12 @@
 +++
 fragment = "footer"
-#disabled = true
 date = "2016-09-07"
```

Comment:
I think removing date is a better choice. We update files a lot and since we're enabling `gitInfo` it's really not needed.

Reply:
I'll open a general issue for removing the date. Thanks